Gemfile.lock

```diff
@@ -338,6 +338,7 @@ PLATFORMS
   arm64-darwin-21
   arm64-darwin-22
   arm64-darwin-23
+  ruby
```

Comment:
This feels like an odd add 😅 Do you know what caused this?

Reply:
no.... 😬

Reply:
Deleted, let's see what happens 😅